acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/pageobjects/OAuthAuthorizeTokenPage.java

```diff
@@ -27,6 +27,10 @@
 
     private final String requestToken;
 
+    public OAuthAuthorizeTokenPage(Jenkins context, URL pageUrl) {
+        this(context, pageUrl, null);
```

Comment:
as we spoke about. See if you can with relative ease get the `requestToken` from the `URL` to ensure that all the tests continue to provide maximum value.